styles/build.js

```diff
@@ -0,0 +1,58 @@
+const StyleDictionary = require('style-dictionary').extend('./sd.config.js');
+
+StyleDictionary.registerFormat({
+  name: 'themable-scss/web',
+  formatter: (tokens) => {
+    const formattedTokens = tokens.allProperties.map((token) => {
+      let tokenName = token.name;
+      if (tokenName === 'sprk-masthead-translate-y')
+        tokenName = 'sprk-masthead-translateY';
+      return `/// ${token.comment} \n$${tokenName}: ${token.value}${
+        token.themable ? ' !default' : ''
+      };`;
+    });
+    return formattedTokens.join('\n');
+  },
+});
+
+StyleDictionary.registerTransform({
+  name: 'size/px-to-rem',
+  type: 'value',
+  matcher(token) {
+    return (
+      token.attributes.category === 'size' &&
```

Comment:
have to convert all our px values to rems before the rem to pt conversion happens for ios

Reply:
Can we put this in a comment in the code?

Reply:
yeah thats a great idea!